src/app/shared/object-list/my-dspace-result-list-element/claimed-search-result/claimed-search-result-list-element.component.ts

```diff
@@ -90,13 +104,58 @@ export class ClaimedSearchResultListElementComponent extends SearchResultListEle
       tap((itemRD: RemoteData<Item>) => {
         if (isNotEmpty(itemRD) && itemRD.hasSucceeded) {
           this.item$.next(itemRD.payload);
+          this.duplicates$ = this.duplicateDataService.findDuplicates(itemRD.payload.uuid).pipe(
```

Comment:
@kshepherd : I believe I located the cause of the error that I was getting.  The "Claimed" search results list appears to load the duplicates here **without** first checking if the feature is enabled.  

That's different behavior from the "Pool" search results (in `pool-search-result-list-element.component.ts`) where the code first checks to see if duplicate detection is enabled before searching for duplicates.

Reply:
Yes, you're right -- just a case of staring at the same code for days and missing this! Thanks, removed and fixed.